descopesdk/src/main/java/com/descope/android/DescopeFlowCoordinator.kt

```diff
@@ -308,14 +317,13 @@ function prepareWebComponent(wc) {
         origin: '$origin',
     }
 
-    wc.addEventListener('ready', () => {
-        if (!wc.bridgeVersion) {
-            flow.onError('Hosted flow uses unsupported web-component SDK version');
-            return
-        }
-        wc.sdk.webauthn.helpers.isSupported = async () => $isWebAuthnSupported
-        flow.onReady();
-    })
+    if (document.querySelector('descope-wc')?.shadowRoot?.querySelector('descope-container')) {
+        flowBridgeIsReady(wc, 'immediate')
+    } else {
+        wc.addEventListener('ready', () => {
+            flowBridgeIsReady(wc, 'event')
```

Comment:
Changed it since the swift-sdk commit, just to stay consistent.

```suggestion
            flowBridgeIsReady(wc, 'listener')
```